APE0.rst

```diff
@@ -0,0 +1,198 @@
+The Astropy Project Bylaws
+==========================
+
+author: The Astropy Governance Working Group: Tom Aldcroft, Matt Craig, Kelle Cruz, Lia Corrales, Steve Crawford, Nicole Foster, Pey Lian Lim, Stuart Mumford, Adrian Price-Whelan, Thomas Robitaille, David Shupe, Brigitta Sipocz, Erik Tollerud
+
+date-created: 2020 July 31
+
+date-last-revised: 2020 July 31
+
+date-accepted: <replace with accepted date>
+
+type: Process
+
+status: Discussion
+
+Abstract
+========
+This APE is a living document that defines the formal fundamental governance process for the Astropy Project. Governance is based around the election of a Coordination Committee. The Coordination Committee has broad authority, which they seek to exercise as rarely as possible by working towards consensus.
+
+
+Detailed description
+====================
+
+The bylaws are described in this section. The initial version is the outcome of several months of discussion by the Astropy Governance Working Group, which aimed to both codification of existing informal governance rules as well as update it with new ideas that are consistent with the philosophy of and vision for the Astropy project.
+
+
+Ultimate Authority: The Astropy Community
+-----------------------------------------
+At its core, the Astropy Project derives its authority from the whole community - developers, users, and even potential participants in the broader astronomical and scientific software community. All have a vested interest in the success of the Project, so all governance in the end belongs to this community in its entirety.  The governance bylaws described below provide levels of delegation of that authority, particularly to the voting members and the Coordination Committee.  The Astropy Project broadly follows the philosophy of do-ocracy: project participants who gain experience and knowledge through sustained community contributions and have demonstrated adherence to core principles of inclusion and consensus-building will be entrusted with increasing levels of authority. At its core, the Project is both by and for its community, so it is the final source of authority and direction.
```

Comment:
In the last sentence, in 'it is the final source' what is 'it'? Make clearer that it is the community (not the project)

Reply:
(moved elsewhere, but agree!)